src/features/workBench.ts

```diff
@@ -219,35 +220,46 @@ export const userCommands = {
 };
 
 export const addCommand = (
-  textToDisplay: string,
-  callback: (uids: string[]) => unknown | Promise<unknown>,
+  args: AddCommandOptions, 
+  extensionAPI: OnloadArgs["extensionAPI"],
   restoreFocus?: true
-) => {
-  const callbackFunction = async () => {
+  ) => {
+    const callbackFunction = async () => {
     const uid = window.roamAlphaAPI.ui.getFocusedBlock()?.["block-uid"];
     const uids = uid
       ? [uid]
       : Array.from(document.querySelectorAll(`.block-highlight-blue`)).map(
           (d) => getUidsFromId(d.querySelector(".roam-block").id).blockUid
         );
-    Promise.resolve(callback(uids)).then(() => {
+    Promise.resolve(args.callback(uids)).then(() => {
       if (restoreFocus && uids.length === 1) {
         focusMainWindowBlock(uids[0]);
       }
     });
   };
-  const display = "(WB) " + textToDisplay;
-  window.roamAlphaAPI.ui.commandPalette.addCommand({
+  const display = "(WB) " + args.label;
+  const options = { 
     label: display,
     callback: callbackFunction,
-  });
+    disableHotkey: args.disableHotkey,
+    defaultHotkey: args.defaultHotkey
+  };
+  extensionAPI.ui.commandPalette.addCommand(options);
   const command = {
     display,
     cmd: callbackFunction,
   };
   _commands.add(command);
-  return () => _commands.delete(command);
-};
+  return () => 
+  {
+    removeCommand(display, extensionAPI);
+    _commands.delete(command);
+  }
+}
+
+export const removeCommand = (label: string, extensionAPI: OnloadArgs["extensionAPI"]) => {
+  extensionAPI.ui.commandPalette.removeCommand({label});
+}
```

Comment:
With big files like this, one tactic I like to do is leave a comment like this:

> This is the only substantive change to the file, the rest just add `extensionAPI` as an argument to `addCommand`. Any other differences I flag below

Reply:
Sorry, I don't follow.
That would be a comment in the code or on the commit?